eng/InstallRuntimes.proj

```diff
@@ -76,14 +76,12 @@
   <!-- Local private build testing -->
   <ItemGroup Condition="$(PrivateBuildTesting)">
     <TestVersions Include="Latest" RuntimeVersion="$(MicrosoftNETCoreAppVersion)" AspNetVersion="$(MicrosoftAspNetCoreAppRefVersion)" />
-    <TestVersions Include="21" RuntimeVersion="$(MicrosoftNETCoreApp21Version)" />
```

Comment:
Are we going to remove 2.1 testing?

Reply:
No. That was only needed to run xunit. This ItemGroup only tests the private builds (5.0).